robottelo/hosts.py

```diff
@@ -71,6 +71,13 @@ def ip_addr(self):
         ipv4, ipv6 = self.execute('hostname -I').stdout.split()
         return ipv4
 
+    def setup(self):
+        self.remove_katello_ca()
+        self.execute('subscription-manager clean')
+
+    def teardown(self):
+        self.unregister()
+
```

Comment:
rebasing should bring in the current implementations of these
https://github.com/SatelliteQE/robottelo/blob/master/robottelo/hosts.py#L78-L88